oak_attestation_verification/src/verifier.rs

```diff
@@ -120,7 +149,43 @@ fn verify_dice_chain(evidence: &Evidence) -> anyhow::Result<()> {
         verifying_key = cose_key_to_verifying_key(&cose_key).map_err(|msg| anyhow::anyhow!(msg))?;
     }
 
-    Ok(())
+    // Verify application keys in evidence and extract public keys. The verifying key comes from the
+    // last DICE layer.
+    let appl_keys = evidence
+        .application_keys
+        .as_ref()
+        .ok_or_else(|| anyhow::anyhow!("no application fields in evidence"))?;
+
+    // Process encryption certificate.
+    let ecert = coset::CoseSign1::from_slice(&appl_keys.encryption_public_key_certificate)
```

Comment:
In the rest of the code base we try to use more explicity naming (e.g. `encryption_cert`). This naming scheme seems more like Go naming than the Rust naming we have been using so far.

Reply:
Thanks. For me it is the opposite. When I am scanning through a lot of code it is much more difficult to see the difference between `ecert` and `scert` than `encryption_cert` and `signing_cert`.